QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -18,14 +17,14 @@ public static BooleanStatus create() {
 		a.setShouldBeRunning(b);
 		a.updateWhen(Igneous.globalPeriodic);
 		
-		Elevator.elevatorControl.setFalseWhen(BooleanMixing.onRelease(b));
+		Elevator.raising.setFalseWhen(BooleanMixing.onRelease(b));
 		
```

Comment:
Lowering should be set to true here, so that the elevator isn't holding up unnecessary weight